src/iuse.h

```diff
@@ -35,7 +35,6 @@ class iuse
     int ecig                ( player*, item*, bool, const tripoint& );
     int antibiotic          ( player*, item*, bool, const tripoint& );
     int eyedrops            ( player*, item*, bool, const tripoint& );
-    int fungicide           ( player*, item*, bool, const tripoint& );
```

Comment:
That does rather remove the reason for the antifungal drug. The fungal infection is a horrible design - perhaps we could redesign it here?